common/constants.js

```diff
@@ -0,0 +1 @@
+module.exports = { nameMapFile: 'names-from-templates.json' };
```

Comment:
Let's just move this into the `bin/cli.js` file; it's not buying us anything to have a one-line module over here (except some small amount of overhead from `require`ing it!).

Reply:
I hate this file, because it only has one constant in it. 😭

However, at least the stage 4 transform will also need it; that's how it ended up in `common/`. I wanted to make sure there was no way for the file produced by stage 2 and the file read by stage 4 to be different files.

Reply:
Yeah, that makes sense: insofar as you're breaking apart the stages into separate modules, do as you will; but if they're in a single module, just with different functions, no need for a separate module for this!